watson_developer_cloud/discovery_v1.py

```diff
@@ -43,7 +43,7 @@ class DiscoveryV1(WatsonService):
     VERSION_DATE_2017_06_25 = '2017-06-25'
     VERSION_DATE_2016_12_01 = '2016-12-01'
 
-    def __init__(self, version, url=default_url, username=None, password=None):
+    def __init__(self, version, url=default_url, username=None, password=None, http_config={}):
```

Comment:
I'd like to suggest an alternate solution -- just add a method to the `watson_service` class for setting the http_config.  This is probably needed anyway, and once we have it, we don't really need to add an `http_config` parameter to all the service constructors, or change the generator -- it's a much simpler solution.

Reply:
That sounds like a good idea.